sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/CommunicationIdentityAsyncTests.java

```diff
@@ -473,4 +473,19 @@ public void issueTokenWithResponseUsingManagedIdentity(HttpClient httpClient) {
             })
             .verifyComplete();
     }
+
+    private CommunicationIdentityAsyncClient setupAsyncClientUsingManagedIdentity(HttpClient httpClient, String testName) {
+        CommunicationIdentityClientBuilder builder = getCommunicationIdentityClientBuilderUsingManagedIdentity(httpClient);
+        return addLoggingPolicy(builder, testName).buildAsyncClient();
+    }
+
+    private CommunicationIdentityAsyncClient setupAsyncClientUsingConnectionString(HttpClient httpClient, String testName) {
+        CommunicationIdentityClientBuilder builder = getCommunicationIdentityClientUsingConnectionString(httpClient);
+        return addLoggingPolicy(builder, testName).buildAsyncClient();
+    }
+
+    private CommunicationIdentityAsyncClient setupAsyncClient(HttpClient httpClient, String testName) {
+        CommunicationIdentityClientBuilder builder = getCommunicationIdentityClient(httpClient);
+        return addLoggingPolicy(builder, testName).buildAsyncClient();
+    }
```

Comment:
If there was just one `setupAsyncClient` method that took `CommunicationIdentityClientBuilder` and `testName` as args, then you can remove the duplication.

```java
private CommunicationIdentityAsyncClient setupAsyncClient(CommunicationIdentityClientBuilder  builder, String testName) {
    return addLoggingPolicy(builder, testName).buildAsyncClient();
}
```

The other option is to update the `@Test` methods inline:

```java
addLoggingPolicy(getCommunicationIdentityClientBuilderUsingManagedIdentity(httpClient), testName).buildAsyncClient();
```